File: test/fixtures/simple.py
# Simple Python fixture for DAP proxy end-to-end testing.
# Used with debugpy to test breakpoint, step, and inspect operations.

def add(a, b):
    result = a + b
    return result

def main():
    x = 42
    y = add(x, 1)
    print(f"Result: {y}")
    return y

if __name__ == "__main__":
    main()

File: test/fixtures/simple.js
// Simple Node.js fixture for CDP end-to-end testing.
// Used with node --inspect to test breakpoint, step, and inspect operations.

function add(a, b) {
    const result = a + b;
    return result;
}

function main() {
    const x = 42;
    const y = add(x, 1);
    console.log(`Result: ${y}`);
    return y;
}

main();
